libs/langchain/langchain/agents/agent_iterator.py

```diff
@@ -282,7 +282,7 @@ def __next__(self) -> dict[str, Any]:
             return self._call_next()
         except StopIteration:
             raise
-        except (KeyboardInterrupt, Exception) as e:
+        except (KeyboardInterrupt, Exception, BaseException) as e:
```

Comment:
think the others inherit from BaseException so could just do that

Reply:
Oho my friend